strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/BrowseController.java

```diff
@@ -97,10 +103,10 @@
                                      .body("{ 'error': 'The requested storage was not found.' }");
             }
             
-            Map<String,SortedSet<String>> repos = new HashMap<>();
-            repos.put("repositories", new TreeSet<String>(storage.getRepositories().keySet()));
+            Path dirPath = Paths.get(storage.getBasedir());
```

Comment:
same as above.
I think that we should fetch storages and repositories form configuration.

Reply:
+1